scripts/indexer/index.js

```diff
@@ -1,120 +1,59 @@
-require('dotenv').config();
-
-const _ = require('lodash');
-const fastJson = require('fast-json-stringify');
-const stringify = fastJson({
```

Comment:
@Akshat111111 Any reason you've removed `fast-json-stringify`? 

That led to a significantly marked improvement in JSON parsing performance when I tried it with and without on this 32M recordset?